packages/directory/addon/components/dir-table.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright 2018, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ * {{dir-table
+ *   items=items
+ *   isSearching=isSearching
+ * }}
+ */
+import Component from '@ember/component';
+import { computed, get, getWithDefault } from '@ember/object';
+import layout from '../templates/components/dir-table';
+import Table from 'ember-light-table';
+import Moment from 'moment';
+
+export default Component.extend({
+  layout,
+
+  /**
+   * @property {String} tagName
+   */
+  tagName: '',
+
+  /**
+   * @property {Array} model - Used by ember-light-table to create rows
+   */
+  model: computed('items', function() {
+    return getWithDefault(this, 'items', []).map(item => {
+      let type = get(item.serialize().data, 'type');
+
+      return {
+        type,
+        id: get(item, 'id'),
+        name: get(item, 'title'),
+        lastUpdatedDate: Moment(get(item, 'updatedOn')).format('MM/DD/YYYY -  hh:mm:ss a z'),
+        author: get(item, 'author.id')
+      };
+    });
+  }),
+
+  /**
+   * @property {Array} columns - Used by ember-light-table to define each column
+   */
+  columns: computed(function() {
```

Comment:
why computed is needed?

Reply:
That's what the example on the ember-light-table docs used. I will remove it

Reply:
Nevermind. This is again related to leaking state into ember objects. 

`error  Only string, number, symbol, boolean, null, undefined, and function are allowed as default properties  ember/avoid-leaking-state-in-ember-objects`